dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -1554,7 +1561,8 @@
       "description": "Enumeration of supported analysis tasks on a collection of conversations.",
```

Comment:
singular, since i presume you don't give the sentiment across conversations

Reply:
Good catch!